python/ray/data/preprocessors/scaler.py

```diff
@@ -54,17 +54,56 @@ def __repr__(self):
 
 
 class MinMaxScaler(Preprocessor):
-    """Scale values within columns based on min and max values.
+    r"""Scale each column by its range.
 
-    For each column, each value will be transformed to
-    ``(value - min) / (max - min)``,
-    where ``min`` and ``max`` are calculated from the fitted dataset.
+    The general formula is given by
 
-    When transforming the fitted dataset, transformed values will be in the
-    range [0, 1].
+    .. math::
+
+        x' = \frac{x - \min(x)}{\max{x} - \min{x}}
+
+    where :math:`x` is the column and :math:`x'` is the transformed column. If
+    :math:`\max{x} - \min{x} = 0` (i.e., the column is constant-valued), then the
+    transformed column will get filled with zeros.
+
+    Transformed values are always in the range :math:`[0, 1]`.
+
+    .. note::
+        This can be used as an alternative to :py:class:`StandardScaler`.
```

Comment:
nit: Any guidance on when to choose one or the other? Also would it make sense to include other scalers here as well?

Reply:
> Any guidance on when to choose one or the other?

I'm not aware of any.

> Also would it make sense to include other scalers here as well?

Maybe. AFAIK `StandardScaler` and `MinMaxScaler` are the two general-purpose scalers. The other scalers are intended for specific types of data.